### ISSUE TITLE
Removal of label_studio/frontend/dist from repository

### ISSUE THREAD

**Issue by apohllo** [heartexlabs/label-studio#1321]
**Is your feature request related to a problem? Please describe.**
I spent a lot of time trying to deploy latest version of Labelstud.io. Recently the problem with NER was resolved (i.e. the first token in a line could not be tagged). So I wanted to deploy the latest version of LS. Yet even though the feature worked on local machine, it wasn't working on the production machine.

After long investigation, I figured out that the problem was due to wrong version of the frontend JS files (occupying `label_studion/frontend/dist` directory). The files on the local machine and on the production server were different. I looked into the history of the directory and it turns out it was updated several times yesterday. And probably this is the reason for the differences in working of the frontend.

**Describe the solution you'd like**
I would consider removing the `dist` directory (its subdirectories or the generated JS files) from the git repository and force a build of frontend during the build of the image of label studio `app` service. Currently, the frontend is not build (thanks to `prebuild_wo_frontend.sh`). I don't know the rationale for that decision, yet it seems strange that during the build the important part of the application is not rebuild. As a result this part might be stale and may cause hard to understand deployment issues.

**Describe alternatives you've considered**
Removal of the directory might not be necessary, but I think that rebuild of the JS files is a must.

**Additional context**
Have you considered deploying the official images of LS to public Docker repository? This would make deployment of the specific version of the application much easier. The user would have to provide the version of LS for docker-compose (e.g. via env variable) and the image would be pulled from the repository. Such a solution would be very robust, assuming the images would be first verified to work correctly by the deployment team.

**Comment by apohllo** [comment]
I can answer myself partly - there's a public repository with LS images:
https://hub.docker.com/r/heartexlabs/label-studio

So I guess this is the preferred, authoritative source of official LS builds. Maybe I would make sense to mention that in the readme?

**Comment by makseq** [comment]
@apohllo 
You can fetch LSF and DM using the following command:
```
make frontend-fetch
```

Also you can build react-app (main LS pages like organization, projects, ...) with:
```
make frontend-build
```

https://hub.docker.com/r/heartexlabs/label-studio --- yes, it's official repo. I've added a link to it here: https://github.com/heartexlabs/label-studio/pull/1325

**Comment by apohllo** [comment]
@makseq thank you!

**Comment by apohllo** [comment]
I'm wondering, why building the frontend is not a part of the Docker image rebuild...

**Comment by makseq** [comment]
@apohllo Do you mean - why don't we build frontend on the docker pre-build step?

**Comment by apohllo** [comment]
Yes, exactly. Maybe there should be a separate script for building custom production images?
I suppose that building docker image is mainly part of the development process and I guess many of the backend developers don't need the frontend, so they don't have npm installed. Is that right?

Anyway to build a fully functional production (custom) image we have to:
```
make frontend-fetch
make frontend-build
docker-compose up --rebuild --forece-recreate
```

Not a big deal TBH, so maybe it's just the matter of documentation.
